app/models/dish.rb

```diff
@@ -1,4 +1,10 @@
 class Dish < ApplicationRecord
   validates_presence_of :name, :description
   belongs_to :chef
+  has_many :dish_ingredients
+  has_many :ingredients, through: :dish_ingredients 
+
+  def calorie_count
+    ingredients.sum(:calories)
+  end
```

Comment:
Yes, great job with this method, however, i didn't see a test in the model spec. Make sure we test our model methods.